thieme/journals.tab

```diff
@@ -0,0 +1,79 @@
+TITLE	IDENTIFIER	ISSN	EISSN
+AINS	ains	0939-2661 	1439-1074
```

Comment:
@damnation333, identifiers are automatically generated from the title, so no need to include them here. (I'll make the change)